src/connection/passwordless/email_sent_confirmation.jsx

```diff
@@ -16,7 +16,7 @@ class ResendLink extends React.Component {
   render() {
     const { label, onClick } = this.props;
     return (
-      <a className="auth0-lock-resend-link" href="javascript:void(0)" onClick={onClick}>
+      <a className="auth0-lock-resend-link" href="#" onClick={onClick}>
```

Comment:
Should we handle the `event.preventDefault` internally? As I think this might end up in changed behavior if we dont, we can not guarantee the caller calls event.preventDefault (also previously they didnt need to)

Reply:
Yes we do already. The component that uses this (`Resend`) prevents that default click in its own handler, and is the only thing that uses this `ResendLink` component (it's not exported anywhere).

Reply:
Oh, I missed that!